### PR TITLE
FOGL-7510: Fixed error handling issue for forkJoin on north & south modal window

### PR DESCRIPTION
**Pull request by mshariq-nerd**
**NOTE:**  Added open source library for alert messages handling [bulma-toast](https://github.com/rfoel/bulma-toast). Plan is to deprecate alert-component in favour of bulma-toast.

### CODE REVIEW

**Review by praveen-garg — COMMENTED** [review]
What if there is only one request and that got an error?

When we click and there is nothing to Save, We should probably show an info alert "Nothing to save"

**Comment by mshariq-nerd** [comment]
> What if there is only one request and that got an error?
There will be only one error alert

File: src/app/components/core/south/south-service-modal/south-service-modal.component.ts [hunk 1]
```diff
@@ -580,6 +580,7 @@ export class SouthServiceModalComponent implements OnInit {
     if (this.apiCallsStack.length > 0) {
       this.ngProgress.start();
       forkJoin(this.apiCallsStack).subscribe((result) => {
+        console.log('result', result);
```
Comment (monika-Dianomic) [inline src/app/components/core/south/south-service-modal/south-service-modal.component.ts]: please remove

File: src/app/components/core/north/north-task-modal/north-task-modal.component.ts [hunk 2]
```diff
@@ -534,35 +539,50 @@ export class NorthTaskModalComponent implements OnInit, OnChanges {
...
 
     this.apiCallsStack.push(this.configService.
-      updateBulkConfiguration(categoryName, configuration).pipe(catchError(e => of(e))));
+      updateBulkConfiguration(categoryName, configuration)
+      .pipe(map(() => ({ type, success: true })))
+      .pipe(catchError(e => of({ error: e, failed: true }))));
   }
 
   save() {
+    console.log('save');
```
Comment (praveen-garg) [inline src/app/components/core/north/north-task-modal/north-task-modal.component.ts]: remove?

File: src/app/components/core/north/north-task-modal/north-task-modal.component.ts [hunk 3]
```diff
@@ -534,35 +539,50 @@ export class NorthTaskModalComponent implements OnInit, OnChanges {
...
+            this.response.handleResponseMessage(r.type);
+          }
+        });
         this.notify.emit();
         this.toggleModal(false);
         this.form.reset();
         this.apiCallsStack = [];
       });
     } else {
+      this.toast.info('Nothing to save', 5000);
```
Comment (praveen-garg) [inline src/app/components/core/north/north-task-modal/north-task-modal.component.ts:583]: 3000 is enough

**Review by monika-Dianomic — APPROVED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Pull request merged**